src/components/ble/MusicService.h

```diff
@@ -25,6 +25,7 @@
 #include <host/ble_uuid.h>
 #undef max
 #undef min
+#include "portmacro_cmsis.h"
```

Comment:
`portmacro_cmsis.h` is a header internal to FreeRTOS. It would be better to just include `FreeRTOS.h`. Also, we use `<>` for external libraries like this rather than `""`

Reply:
thanks for the review. Indeed `<>` for external libraries. Review are IMPORTANT. Thanks for the time and suggestion to make it right